aodntools/timeseries_products/Documentation/aggregated_timeseries.md

```diff
@@ -0,0 +1,175 @@
+- PSAL (sea_water_salinity);
+- CPHL, CHLF, CHLU (mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water) - considered as separate variables for now;
+- TURB (sea_water_turbidity);
+- DOX1, DOX1_2, DOX1_3 (mole_concentration_of_dissolved_molecular_oxygen_in_sea_water);
+- DOX2 (moles_of_oxygen_per_unit_mass_in_sea_water);
+- DOXS (fractional_saturation_of_oxygen_in_sea_water).
+- PAR (downwelling_photosynthetic_photon_flux)
+
+## Method
+
+Generating function: 
+
+```
+usage: aggregated_timeseries.py [-h] -var VARNAME -site SITE_CODE -files
+                                FILENAMES [-path OUTPUT_PATH]
+
+Concatenate ONE variable from ALL instruments from ALL deployments from ONE
+site
+
+optional arguments:
+  -h, --help         show this help message and exit
+  -var VARNAME       name of the variable to concatenate. Like TEMP, PSAL
+  -site SITE_CODE    site code, like NRMMAI
+  -files FILENAMES   name of the file that contains the source URLs
+  -path OUTPUT_PATH  path where the result file will be written. Default ./
+
+```
+
+
+
+
+### Input file selection
+
+The input files are accessed using the OPeNDAP protocol via the AODN THREDDS server.  A list of OPeNDAP URLs can be produced by the utility function get_moorings_url(), which queries an index of all published moorings files to obtain the files relevant for a given site and variable of interest (VoI). The list can also be filtered by feature type (e.g time-series or profile) temporal range, file version (FV01), processing mode (real-time/delayed) or a regular expression matched against the file name.
+
+### Input file validation
+
+Before proceeding to the aggregation, each input file will be checked to ensure it meets the requirements (as specified above under Inputs). This will prevent errors during the aggregation process. If any files fail to meet the requirements, a list of all non-compliant files is output, and the code stops. An option can be provided to continue processing only the compliant files.
+
+### Dimensions
+
+The dimensions of the resulting file  are determined as follows:
+`OBSERVATION`:  the total number of observation records, excluding out-of-the-water data, in all input files;
+`INSTRUMENT`: the number of instruments (i.e. number of files);
+`string256`: a fixed dimension of length 256 for character array variables.
+
+### Variables
+
+The VoI is produced by sequentially concatenating the individual values in each of the input files. The resulting variable has dimension `OBSERVATION`. The VoI’s ancillary_variables, in particular the corresponding quality-control flags, are also included, with dimension OBSERVATION. If the quality control variable is absent in any input file, the corresponding flags in the output file will  be set to 0 (“no QC performed”) and a warning will be raised.
+
+The variable `TIME` from input files is concatenated into a variable TIME(OBS). This could result in a non-uniform time interval.
```

Comment:
OBS -> OBSERVATION, and again two more times in the next paragraph.

Reply:
already fixed